api4/system_test.go

```diff
@@ -67,23 +68,17 @@ func TestGetAudits(t *testing.T) {
 	audits, resp := th.SystemAdminClient.GetAudits(0, 100, "")
 	CheckNoError(t, resp)
 
```

Comment:
Please remove the empty line between `CheckNoError` and the `require` checks here and in the following lines.